packages/core/src/network.js

```diff
@@ -264,6 +292,7 @@ async function sendResponseResource(network, request, session) {
   let log = network.log;
   let url = originURL(request);
   let meta = { ...network.meta, url };
+  let send = async (method, params) => await network.send(session, method, params);
```

Comment:
Optional
```suggestion
  let send = (method, params) => network.send(session, method, params);
```

Reply:
Just reduction in number of params, code looks worse to read [ and more off from how it looks for `session.send` calls ]